x-pack/test_serverless/functional/test_suites/common/examples/data_view_field_editor_example/index.ts

```diff
@@ -55,7 +55,15 @@ export default function ({ getService, getPageObjects, loadTestFile }: FtrProvid
         }
         return false;
       });
-      await PageObjects.settings.createIndexPattern('blogs', null);
+      // re-create default dataview : default_all_data_id which was created when serverles_search plugin was started.
+      await PageObjects.settings.createIndexPattern(
```

Comment:
_Context_: As this test begins, it cleans up the default dataview  that was created when `serverless_search` had started.

Line where deletion happens  https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/functional/test_suites/common/examples/data_view_field_editor_example/index.ts#L33

As a result, the FTR  test were failing as [data_view_field_editor_example.ts](https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/functional/test_suites/common/examples/data_view_field_editor_example/data_view_field_editor_example.ts) was expecting the default dataview id.  

In this example, instead of creating new a dataview `blogs`, re-creating the default dataview : `default:all-data` with indexPattern set to `blogs`, (although `serverless_search `maps to indexPattern `* `).  Figured it should be fine.